go.mod

```diff
@@ -55,8 +54,8 @@ require (
 	gopkg.in/resty.v1 v1.12.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	gorm.io/driver/postgres v1.0.8
-	gorm.io/gorm v1.21.7
+	gorm.io/driver/postgres v1.4.6
+	gorm.io/gorm v1.24.3
```

Comment:
I saw there is a new release: https://github.com/go-gorm/gorm/releases/tag/v1.24.6

Reply:
See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1637#discussion_r1154569175